File: in-motion-server/Post/IMS.Post.Models/Dto/Outgoing/PostVideoDto.cs
namespace IMS.Post.Models.Dto.Outgoing;

public class PostVideoDto
{
    public string Id { get; set; }
    public string Filename { get; set; }
    public string BucketName { get; set; }
    public string BucketLocation { get; set; }
    public string ContentType { get; set; }
    public string VideoType { get; set; }
}

File: in-motion-android/app/src/main/java/com/inmotion/in_motion_android/data/remote/dto/posts/PostVideoDto.kt
package com.inmotion.in_motion_android.data.remote.dto.posts

import java.io.Serializable

data class PostVideoDto (
    val id: String,
    val filename: String,
    val bucketName: String,
    val bucketLocation: String,
    val contentType: String,
    val videoType: String
): Serializable
